go/genkit/servers.go

```diff
@@ -380,12 +380,12 @@ func nonDurableFlowHandler(f flow) func(http.ResponseWriter, *http.Request) erro
 			return err
 		}
 		var callback streamingCallback[json.RawMessage]
-		if stream {
+		if r.Header.Get("Accept") == "text/event-stream" || stream {
 			w.Header().Set("Content-Type", "text/plain")
 			w.Header().Set("Transfer-Encoding", "chunked")
 			// Stream results are newline-separated JSON.
 			callback = func(ctx context.Context, msg json.RawMessage) error {
-				_, err := fmt.Fprintf(w, "%s\n", msg)
+				_, err := fmt.Fprintf(w, "data: %s\n\n", msg)
```

Comment:
this needs to be wrapped in a `{"message":  ...}` object.

Reply:
```suggestion
				_, err := fmt.Fprintf(w, "data: {"message": %s}\n\n", msg)
```

Reply:
Thanks Pavel, adding the changes now

Reply:
I would recommend testing this using the genkit client library:
https://github.com/firebase/genkit/blob/main/js/genkit/src/client/client.ts


```ts
import { streamFlow } from 'genkit/client';

(async () => {
  const response = await streamFlow({
    url: 'http://localhost:3400/myTestFlow',
    input: {foo: 'bar'},
  });

  for await (const chunk of response.stream()) {
    console.log(chunk)
  }
  console.log(await response.output())
})()
```